src/services/platform/subscription/subscription.events.ts

```diff
@@ -0,0 +1,3 @@
+export const COMMUNICATION_MESSAGE_RECEIVED = 'communicationMessageReceived';
+export const ROOM_INVITATION_RECEIVED = 'roomInvitationReceived';
```

Comment:
I see other MATRIX specific constants prefixed with MATRIX, maybe MATRIX_ROOM_INVITATION_RECEIVED to be consistent?

Reply:
These are events visible on our api, so they should not expose Matrix. The ROOM _INVITATION_RECEIVED should I think have a COMMUNICATION prefix for consistency.